### DISCUSSION TITLE
Ender 3 w/ CR Touch. Board: V4.2.2 GD32 RET6 chip.

### DISCUSSION THREAD

**Discussion by Stemwrench** [mriscoc/Ender3V2S1#950] [Q&A]
Is there a compatible firmware for and Ender 3 with CR Touch with a V4.2.2 motherboard and GD32 RET6 chip? If so, can ya'll point me to the right direction. 

Thanks

**Comment by mill1000** [comment]
Maybe see #413. It's for the V2 Neo but there's discussion of GD32 chips.

**Comment by classicrocker883** [comment]
GD32 should work the same `default_envs` as STM32F103RE_creality_maple

**Reply by mriscoc** [reply]
Some people have had successful with the current STM32F103RE_creality environment.

**Reply by Stemwrench** [reply]
Yea. I’ve flashed it and it was successful and also flashed my Ender 3 V2 as well. Both as success. Also, was looking into flashing the screen but have not seen  or heard anyone flashing a VIEWE lcd screen that came with my Ender 3 V2.

**Reply by mriscoc** [reply]
SYNWIT (VIEWE) display, initially unsupported due to incompatibilities with QR-codes (removed in latest firmware version) some users [reported problems](https://github.com/mriscoc/Ender3V2S1/issues/323) with this screen, but now seems that it is working without issues. Some users can even see a G-code preview.

**Reply by Stemwrench** [reply]
Yeah, that’s why I didn’t bother flashing it. The new features look great but as long as I’m able to use my screen, there’s no need to flash it.

**Reply by Henk045** [reply]
Maybe a real dumb question. But where do I find the files for the Synwit (viewe) displays? I've tryd the DWIN_SET and the private folder aswell. Both gave me a blue screen error with no boot to be found. Sorry if this question has been asked. But I've tryd to do my researche but can't figure it out after a couple of hours. ( yes i've watched youtube vids, and browsed on reddit and github.) This is why it is confusing me, i thought you meant it's supported now ( " , initially unsupported due to incompatibilities with QR-codes (removed in latest firmware version) some users (link) with this screen, but now seems that it is working without issues " ) Meaning it should work right? ( trying to get a firmware for the display without any addons )

**Reply by mriscoc** [reply]
The SYNWIT display can be used with the latest firmware version; no display firmware update is required. There is currently no update available for the SYNWIT display.

**Reply by Henk045** [reply]
Thank you very much for the really fast reply! I appricate your work and time you put into it!! I'm using the latest version. Just have to get used to it. And see what features you managed to make! Again. Thank you very much for the fast reply! 

Stay safe! And thank you again.
